worker/src/RTC/SvcConsumer.cpp

```diff
@@ -423,8 +423,7 @@ namespace RTC
 					continue;
 				}
 
-				requiredBitrate =
-				  this->producerRtpStream->GetLayerBitrate(nowMs, spatialLayer, temporalLayer);
+				requiredBitrate = this->producerRtpStream->GetBitrate(nowMs, spatialLayer, temporalLayer);
```

Comment:
I'm looking at this and I don't understand the change. Here we need to compute the bitrate needed to upgrade to next layer so what we need to obtain is the bitrate of the next layer (the desired layer) rather than the total bitrate of the desired layer plus all layers below. This is the IncreaseLayer() method which means "hey take this available bitrate and use it to upgrade one layer and tell me how much bitrate you needed to upgrade to that layer".

NOTE: Not 100% sure of what I'm saying.

Reply:
We have 3 spatial and 2 temporal layers.
so 2:1 is the highest. It must be a sum. i.e. 2:0 bitrate is 2.9 Mbps and 2:1 is 1.9 Mbps, but when switching to 2:1 we need to account for 4.8Mbps but mediasoup was accounting for 1.9 Mbps only which was leading to this issue

Reply:
NOTE: I am also already not 100% sure about that.

Reply:
> 2:0 bitrate is 2.9 Mbps and 2:1 is 1.9 Mbps, but when switching to 2:1 we need to account for 4.8Mbps 

This is what the change in the PR does but this is exactly what I mean is wrong. This consumer already used the 2.9MB of the 2:0 layer in the previous call to IncreaseLayer() so now that it upgrades to 2:1 it needs to use 1.9MB and that's what the method has to return now. In the previous call it already used and returned 2.9MB. I don't think this was wrong at all (note: I've taken a look yet to changes in SimulcastConsumer).

Reply:
@ibc Seems like you are right, more over, i did tests, and that is not fixing an issue((( Only makes it worse.

Reply:
Yep, definitely this PR is not OK. We are correctly increasing layer by layer already.